### ISSUE TITLE
data-nodrag not working?

### ISSUE THREAD

**Issue by mikeRead** [angular-ui-tree/angular-ui-tree#433]
My setup is just like the docs with "Unlimited nesting HTML View or Templates Example" however I can not delete or collapse unless i click multiple times as fast as I can.When I click on the "delete" buttton I can still drag the item around. Also ```79:  nodrag: function (targetElm){ ```` does not get called to check if data-nodrag is set, Any idea why? I have below in my controller
```javascript 
$scope.remove = function(scope) {
      scope.remove();
    }; 
```

I need to use a AngularJS v1.3.15 the demo is in v1.2.15 could this be the issue?

thanks

**Comment by jberzins** [comment]
There is some defect in component version 2.2.0
I took latest release 2.1.5 and it works.

**Comment by mikeRead** [comment]
That worked great, I thought I was going crazy,  thanks so much!

**Comment by enkodellc** [comment]
Check out my branch has several fixes and a couple new features. Since @JimLiu has not been responsive we might want to fork for good and get a few more people to manage the repo.

https://github.com/enkodellc/angular-ui-tree

**Comment by Voles** [comment]
This issue has been fixed with #411 by @enkodellc.
